activiti-core/activiti-api-impl/activiti-api-process-runtime-impl/src/test/java/org/activiti/runtime/api/impl/ExtensionsVariablesMappingProviderTest.java

```diff
@@ -1092,6 +1092,21 @@ public void should_substituteExpressions_when_customExpression() throws Exceptio
             );
     }
 
+    @Test
+    public void shouldReadFieldEphemeralWhenSetInExtension() throws Exception {
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Instantiating `ObjectMapper` is considered as expensive, so it's better to use a static variable for it.